README.md

```diff
@@ -9,7 +9,11 @@ To add a conference to the list, please [contribute](#contributing)!
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [PyCon India 2019](https://in.pycon.org/2019/) |  12-15 October |  Chennai Trade Center and IIT Madras Research Park  | India's premier conference on using and developing the Python programming language | No |
-| [JSFoo Bangalore] (https://jsfoo.in/2019/) | 27-28 Sep | Nimhans Convention Center, Bangalore | JSFoo is in its ninth edition this year. The 2019 edition is a two-track event with talks and Birds of Feather (BOF) sessions | No |
+| [JSFoo Bangalore](https://hasgeek.com/jsfoo/2019/) | 27-28 Sep | NIMHANS Convention Centre, Bangalore | JSFoo is in its ninth edition this year. The 2019 edition is a two-track event with talks and Birds of Feather (BOF) sessions | No |
+| [Rootconf 2019](https://hasgeek.com/rootconf/2019/) | 21-22 Jun | NIMHANS Convention Centre, Bangalore | India's largest community and conference on infrastructure security, DevOps and distributed systems | Yes, for speakers |
```

Comment:
nit: can you remove the 2019 from all the conference titles? It's supposed to be implicit :stuck_out_tongue:

Reply:
> Thanks a **lot**, Abhishek! :)
> 
> This repo only accepts conferences which have CFPs and Code of Conducts. I can't find CoCs for these conferences...do you think it'd be possible to add a CoC for each of them (I see that you work at HasGeek)? Happy to merge this PR after that. 👍

Sure, just wanted to clarify that we have a common code of conduct across HasGeek events. https://hasgeek.com/about/policy/code

We also have standees at events which display the code of conduct and contact information prominently around the event venue.

Reply:
Thanks for the info, that's good enough to get this merged.

Would definitely be nicer to have it linked on each conference's website too though. :grimacing: